docs/source/overview.rst

```diff
@@ -0,0 +1,32 @@
+Overview
+========
+
+What is Kubeflow Pipelines?
+----------------------------
+
+.. raw:: html
```

Comment:
not sure if this is needed since there's no embeded html here

Reply:
oh good catch.